src/endpoint/glossary/languages.rs

```diff
@@ -0,0 +1,80 @@
+macro_rules! impl_glossary_languages {
+    ( $($lang:ident, $name:literal, $lower:literal $(,)? )+  ) => {
```

Comment:
I think we can make use of https://docs.rs/paste/latest/paste/#case-conversion instead of handwritten here